lib/awsutil/api.go

```diff
@@ -1,20 +1,48 @@
 package awsutil
 
 import (
+	"flag"
+
 	"github.com/Symantec/Dominator/lib/log"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
+var (
+	awsConfigFile = flag.String(
+		"awsConfigFile", "", "Location of AWS config file")
+	awsCredentialsFile = flag.String(
+		"awsCredentialsFile", "", "Location of AWS credentials file")
+)
```

Comment:
As I said earlier, I'd prefer that the flag default values come from the environment (i.e. `AWS_CONFIG`) if set, else the default would be under the home directory.

Reply:
Yes, I thought about doing it the way you propose, but the environment table is mutable (os.Setenv). So I elected to look at the environment table each time rather than once when flag.Parse() is called.  That way the library honours calls to os.Setenv().

That said, I am happy to make the requested change, but keep in mind that the library will always use the value of AWS_CONFIG_FILE when flag.Parse() was called rather than whatever the current value of AWS_CONFIG_FILE happens to be.